Apidaze.SDK.Tests.Integration/UpdateExternalScriptUrl/Program.cs

```diff
@@ -35,14 +35,14 @@ static void Main(string[] args)
             var id = 1801L;
 
             // new url
-            var newScriptUrl = "http://";
+            var newScriptUrl = "http://.ngrok.io";
```

Comment:
Same here, no ngrok please. Please check if your code has reference to ngrok somewhere

Reply:
ok.